compiler/luci-micro/CMakeLists.txt

```diff
@@ -48,6 +56,30 @@ add_custom_command(
 
 add_custom_target(luci_interpreter_micro_arm_cmake DEPENDS "${MICRO_ARM_BUILD_DEPENDENCY}")
 
+# Generate KernelsToBuild list from circle model
+if (GENERATE_KERNELS_LIST_FROM)
+  if(NOT TARGET mio_circle04)
+    return()
+  endif(NOT TARGET mio_circle04)
+
+  add_executable(generator_kernels_list_exec helpers/GenerateKernelsListHelper.cpp)
+  target_link_libraries(generator_kernels_list_exec arser)
+  target_link_libraries(generator_kernels_list_exec foder)
```

Comment:
If we move interpreter outside of the compiler directory, this could be an issue